### DISCUSSION TITLE
Use Subscriptions in Checkout Pro

### DISCUSSION THREAD

**Discussion by cauexotico** [mercadopago/sdk-js#163] [Ideas]
Is it possible to use Subscriptions (preapproval) with the checkout Pro? 

If not, the ability to set it as preference would be great

**Comment by cauexotico** [comment]
Send a POST request to https://api.mercadopago.com/preapproval_plan returns the init_point, that can be used to make the payment.

**Comment by icaldana** [comment]
Hello @cauexotico!
I am glad you found the answer, thank you for giving our product a trust vote.
